test/common/http/codec_impl_fuzz_test.cc

```diff
@@ -432,6 +439,7 @@ class HttpStream : public LinkedObject<HttpStream> {
       // Maybe nothing is set?
       break;
     }
+    stream_action_active_ = false;
```

Comment:
I think this should be moved to the [end of the `kRequest` case](https://github.com/envoyproxy/envoy/blob/9d8fc1385986e77e325e8e9cc05e36f1082763a9/test/common/http/codec_impl_fuzz_test.cc#L429), as it is easier to understand the relevant scope that this covers.

It would have been great if this could be a local var in the case scope, instead of a data member, but I guess we then we'll have access to a deallocated local var.

Reply:
done.